gemfiles/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_RETRY: "1"
```

Comment:
**question:** As I understand, the default value for this is 3. Any reason why we're setting this to 1?

Reply:
Don't remember honestly, will look up whether this needs to be set at all

Reply:
This is set by Appraisal and I rather not muck about with it too much since it's generated code